src/components/input/TypeHandler/price.ts

```diff
@@ -0,0 +1,65 @@
+import { Base } from "./Base"
+import { Component } from "../Component"
+import { TypeHandler } from "./TypeHandler"
+import { StateEditor } from "../StateEditor"
+import { State } from "../State"
+
+class Price extends Base {
```

Comment:
Price must now the currency to be useful.

Reply:
Do I pass that information down all the way from the SmoothlyInput? Is there a clever way to do that? Adding a Currency prop to SmoothlyInput doesn't feel like a good way, e.g.

Reply:
I hardcoded SEK to start with.

Reply:
OK, we need to add a clever way to do it.

Reply:
I'm sending the rest over for now.